webview-ui/src/comp/Placements.tsx

```diff
@@ -1,44 +1,81 @@
+                <Gallery className='ocm-gallery' hasGutter={true} >
+                { Props.placements.map( (placement) => { 
+                           
+                            const codeJson = placement.kr.spec 
+                            const code =yaml.dump(codeJson);
+                            const row = placement.kr.status.conditions.map( (condition:any) => { 
+                                return [new Date(condition.lastTransitionTime).toLocaleString("en-US",DateFormat),
+                                        condition.message,
+                                        condition.reason,
+                                        condition.status
+                                    ]      
+                                })  
+                            const placementName =  placement.kr.metadata.name;    
+                            const labels = placement.kr.metadata.labels? <OcmLabels labels={placement.kr.metadata.labels} />:null     
+                    return <>
+                            <GalleryTableComponent
+                                title={`Name: ${placementName}`}
+                                subtitle={`Namespace: ${placement.kr.metadata.namespace}`}
+                                rows={row}
+                                code={code}
+                                id={`${placementName}`} > 
+                                <Accordion> 
+                                    <AccordionItem>
+                                        <AccordionToggle  onClick={() => {
+                                                                            onToggle(`def-list-${placementName}`);
+                                                                        }}
+                                                                        isExpanded={expanded === `def-list-${placementName}`}
+                                                                        id={`def-list-${placementName}`}>
+                                            Number of selected clusters: {placement.kr.status.numberOfSelectedClusters} 
+                                        </AccordionToggle>
+                                    <AccordionContent id={`def-expanded-${placementName}`} isHidden={expanded !== `def-list-${placementName}`}>
+                                        <PlacementDecision placementName={placementName}/>
+                                    </AccordionContent>    
+                                    </AccordionItem>
+                                </Accordion>
+                            {labels}     
+                            </GalleryTableComponent>
+                        </>   
                         }
-                    </VSCodeDataGrid>
-                    <div style={{ borderTop: "1px solid #fff ", marginLeft: 10, marginRight: 10 }}></div>
+                    )}                            
+                </Gallery>
                 </>
-            }
+            }            
```

Comment:
There's a VSCode plugin that deletes these type of leftover spaces automatically on-save.

Reply:
name?